recipes/webcolors/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.5" %}
+
+package:
+  name: webcolors
+  version: {{ version }}
+
+source:
+  fn: webcolors-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/w/webcolors/webcolors-{{ version }}.tar.gz
+  md5: 2e9da1bbb7a087c6108e05be64640afb
+
+build:
+  number: 0
+  script:
+    - source activate _build  # [unix]
+    - activate _build         # [win]
```

Comment:
Though I do have a sort of working Windows machine (think the compiler installs didn't go over well 😨). Fortunately, this doesn't require working compilers (I would hope 👼). FWICT it is not activating correctly there. 😕